File: src/main/java/quebec/virtualite/backend/services/domain/DomainService.java
package quebec.virtualite.backend.services.domain;

import quebec.virtualite.backend.services.domain.entities.Greeting;

import java.util.List;

public interface DomainService
{
    void deleteGreetings();

    List<Greeting> getGreetings();

    void recordGreeting(String name);
}

File: src/main/java/quebec/virtualite/backend/services/domain/DomainService.kt
package quebec.virtualite.backend.services.domain;

import quebec.virtualite.backend.services.domain.entities.GreetingEntity;

import java.util.List;

public interface DomainService
{
    void deleteGreetings();

    List<GreetingEntity> getGreetings();

    void recordGreeting(String name);
}
